aws-rollback.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
+
+import json
+import subprocess
+import sys
+
+DEFAULT_REPOSITORY_NAME = "rust-central-station"
+TARGET_TAG = 'latest'
+ECS_CLUSTER = 'rust-ecs-prod'
+ECS_SERVICE = 'TODO COMPLETE'
```

Comment:
A solution here could be to list the services inside the cluster with:

```
aws ecs list-services --cluster {ECS_CLUSTER}
```

If a service with the same name as the repository exists we redeploy that service, otherwise we rollback but *not* redeploy.

Reply:
Sounds good, and it ok to have the `ECS_CLUSTER` as a constant or you want to receive as argument?
Thx!

Reply:
Yep, we only have one cluster at the moment.